HopsanCLI/HopsanCLI.pro

```diff
@@ -69,6 +69,19 @@ CONFIG(debug, debug|release) {
   QMAKE_CXXFLAGS += -pedantic -Wno-long-long -Wconversion
 }
 
+# Set hdf5 paths
+INCLUDEPATH *= $${PWD}/../hopsanhdf5exporter
+LIBS += -L$${PWD}/../lib -lhopsanhdf5exporter$${DEBUG_EXT}
```

Comment:
Move include path and libs into the check below, so that they are only added if have_hdf5() is true

Reply:
Actually this is a problem. The exporter lib needs to be linked before the hdf libs, or there will be undefined references. I will need to think some more about this.

Reply:
But if hdf5 is not available, then they CLI should not link withbthe exporter, because it wont exist.

Reply:
I "solved" it by always adding libs, and then removing it if hdf5 was not found. Kind of ugly, but it works.

Reply:
Aha, I realize now why it did not work, the "have_hdf5" function not only checks, it adds link flags as well. So they end up before hopsanhdf5exporter unless you add the link flags before. I guess we have to live with this solution in the qmake build system.